evap/staff/tests/test_views.py

```diff
@@ -1924,6 +1924,79 @@ def test_evaluation_copy(self):
+        vote_start_datetime = datetime.datetime.now() - datetime.timedelta(days=50)
+        cls.evaluation = baker.make(Evaluation, course=cls.course, vote_start_datetime=vote_start_datetime)
+        cls.contributions = baker.make(
+            Contribution, evaluation=cls.evaluation, _fill_optional=["contributor"], _quantity=3, _bulk_create=True
+        )
+        cls.questionnaire = baker.make(Questionnaire, pk=83)
+        cls.url = reverse("staff:create_exam_evaluation")
+        cls.exam_date = (datetime.date.today()) + datetime.timedelta(days=10)
+
+    def test_create_exam_evaluation(self):
+        self.app.post(
+            self.url,
+            user=self.manager,
+            status=200,
+            params={"evaluation_id": self.evaluation.pk, "exam_date": self.exam_date},
+        )
+        self.assertEqual(Evaluation.objects.count(), 2)
+        exam_evaluation = Evaluation.objects.exclude(pk=self.evaluation.pk).get()
+        self.assertEqual(exam_evaluation.contributions.count(), self.evaluation.contributions.count())
+        self.assertEqual(
+            exam_evaluation.vote_start_datetime,
+            datetime.datetime.combine(self.exam_date + datetime.timedelta(days=1), datetime.time(8, 0)),
+        )
+        self.assertEqual(exam_evaluation.vote_end_date, self.exam_date + datetime.timedelta(days=3))
+        self.assertEqual(exam_evaluation.name_de, "Klausur")
+        self.assertEqual(exam_evaluation.name_en, "Exam")
+        self.assertEqual(exam_evaluation.course, self.evaluation.course)
+        self.assertEqual(exam_evaluation.participants.count(), self.evaluation.participants.count())
+        self.assertEqual(exam_evaluation.weight, 1)
+
+    def test_exam_evaluation_for_single_result(self):
+        self.evaluation.is_single_result = True
+        self.evaluation.save()
+        self.app.post(
+            self.url,
+            user=self.manager,
+            status=400,
+            params={"evaluation_id": self.evaluation.pk, "exam_date": self.exam_date},
+        )
+        self.assertFalse(self.evaluation.has_exam)
+
+    def test_exam_evaluation_for_already_existing_exam_evaluation(self):
+        baker.make(Evaluation, course=self.course, name_en="Exam", name_de="Klausur")
+        self.assertTrue(self.evaluation.has_exam)
+        self.app.post(
+            self.url,
+            user=self.manager,
+            status=400,
+            params={"evaluation_id": self.evaluation.pk, "exam_date": self.exam_date},
+        )
+
+    def test_exam_evaluation_with_wrong_date(self):
+        self.evaluation.vote_start_datetime = datetime.datetime.now() + datetime.timedelta(days=100)
+        self.evaluation.vote_end_date = datetime.date.today() + datetime.timedelta(days=150)
+        self.evaluation.save()
+
+        self.app.post(
+            self.url,
+            user=self.manager,
+            status=200,
+            params={"evaluation_id": self.evaluation.pk, "exam_date": self.exam_date},
+        )
```

Comment:
I think we have a helper to assert that no database modifications were made (`assert_no_database_modifications`). It would probably make sense to use that here